### DISCUSSION TITLE
What is a good known location on MacOS (beginner question)

### DISCUSSION THREAD

**Discussion by Glup3** [neovide/neovide#598] [Q&A] [answered]
Hey guys! I followed the instructions to build Neovide on my machine (MacOS 10.14.6) and I would like to know where I should put the `neovide` file for easy access. I have put it in `/usr/bin/` at the moment and access it with my terminal. Are there any other good locations I should know of?

**Comment by Kethku** [comment, marked as answer]
I converted this to a discussion as it isn't actually an issue with Neovide, and I don't personally have any expertise. Hope thats ok

**Reply by shaunsingh** [reply]
I personally keep it under /Users/(username)/neovide, as I don't have admin permissions 

putting it under /usr/bin is absolutely fine and is what I would recommend

**Reply by josephnoir** [reply]
I think on unix-like systems self-build binaries would go to `/usr/local/bin` rather than `/usr/bin`.

I'm wondering if a valid installation would be to use cargo, e.g., ` cargo install --path . --bin neovide`? This would require require sourcing the cargo environment when starting the terminal as explained when setting up rust via rustup.
